tracee-rules/signatures/rego/anti_debugging_ptraceme.rego

```diff
@@ -1,6 +1,7 @@
 package main
 
 __rego_metadoc__ := {
+    "id": "TRC-1",
```

Comment:
TRC-1 is used for both of this signature and code_injection.go signature - isn't it a problem?

Reply:
Correct - I'll wait to update this PR until #528 gets merged.

Actually you bring up another good point – Sometimes (not in this case) we might have the same signature implemented in both Rego and Go. Do we give the same ID in that case?

Reply:
We will need to find a proper way to track the next available TRC number.
As I mentioned previously, looking forward this will be a point of collisions in the future - think about two open PRs that want to add a new signature - will they both use the same next available number? how will they find it?

Reply:
> Sometimes (not in this case) we might have the same signature implemented in both Rego and Go

I don't think this is a desired state to be in. this will also mean the sig will trigger twice. we shouldn't implement sigs in both

> Do we give the same ID in that case?

I don't think so, it's not the same sig, even if the BL is similar. but again we should never have this case

Reply:
> think about two open PRs that want to add a new signature - will they both use the same next available number? how will they find it?

I honestly don't think this is such a big issue (yet). It's the PR reviewer's responsibility to make sure the number is valid (not just if it's available). and if it's not, to correct the mistake. in rare cases, maybe the reviewer will manually coordinate between 2 open PRs and tell each one which number to use.

Another solution might be to keep a file in the repo that contains the last committed version, and each new sig PR will update it. kind of like a mutex. this can help both easily identify what's the next number and also avoid conflicts.
(it won't prevent 2 people TRYING to use the same number but it will prevent them from succeeding in using it)